celery/__init__.py

```diff
@@ -1,12 +1,11 @@
-# -*- coding: utf-8 -*-
```

Comment:
why did you remove this? It's not related to Python 2

Reply:
# -*- coding: utf-8 -*- ? this? might be mistakenly.

Reply:
No it isn't.
https://www.python.org/dev/peps/pep-0263/

Reply:
https://github.com/pandas-dev/pandas/pull/26197 and django removed them as well

Reply:
Starting with Python 3.0, utf-8 is the default encoding of source files:

https://www.python.org/dev/peps/pep-3120/

This makes the utf-8 encoding cookie redundant.